relay-config/src/config.rs

```diff
@@ -521,6 +513,46 @@ impl Default for Metrics {
             hostname_tag: None,
             buffering: true,
             sample_rate: 1.0,
+        }
+    }
+}
+
+/// Controls processing of metrics and metric metadata.
```

Comment:
```suggestion
/// Controls processing of Sentry metrics and metric metadata.
```
(not sure about this suggestion)